.github/workflows/npm-release.yml

```diff
@@ -0,0 +1,90 @@
+name: NPM Release
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '.gitignore'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '.gitignore'
```

Comment:
@Caknoooo Same as dockerhub, this should release only on release not on every pull request and push to main